sonic/handlers/middlewares.go

```diff
@@ -54,7 +54,7 @@ func AuthMiddleware() gin.HandlerFunc {
 
 		// Parse token
 		jwtToken := authHeader[1]
```

Comment:
It turned out that the sonic pkg uses jwt pkg. Now we don't use sonic as we have stash. Should we delete sonic from pkg?